moped-database/views/project_list_view.sql

```diff
@@ -47,14 +47,14 @@ AS WITH project_person_list_lookup AS (
     mp.interim_project_id,
     string_agg(DISTINCT me2.entity_name, ', '::text) AS project_partner,
     string_agg(task_order_filter.value ->> 'display_name'::text, ','::text) AS task_order_name,
-    (SELECT JSON_AGG(feature.attributes) -- this query finds any signal components and those component's features and rolls them up in a JSON blob
+    (SELECT JSON_AGG(json_build_object('signal_id', feature_signals.signal_id, 'knack_id', feature_signals.knack_id, 'location_name', feature_signals.location_name, 'signal_type', feature_signals.signal_type))
         FROM moped_proj_components components   
-        LEFT JOIN uniform_features feature
-          ON (feature.component_id = components.project_component_id)
+        LEFT JOIN feature_signals
+          ON (feature_signals.component_id = components.project_component_id)
```

Comment:
here's the important change. we used to join up the `uniform_features` view, which itself is a union of all the `feature_signal`, `feature_intersection`, etc... tables. that union is expensive because of limitations around JSON aggregation and i think also because of limitations around table inheritance indexing.

in any case—we're not targeting just the `feature_signals` table which makes our query dramatically faster.

tbd how we will handle adding additional feature tables and feature properties to the list view, if that becomes necessary....